resources/migrations/001_update_migrations.yaml

```diff
@@ -8253,6 +8253,152 @@ databaseChangeLog:
             tableName: login_history
             indexName: idx_user_id_device_id
 
+  - changeSet:
```

Comment:
These (at least the non-sql ones) should probably have `preConditions` or else they'll fail the new lint rule when merging. Would recommend making them idempotent if possible